sandpack-react/src/presets/Sandpack.tsx

```diff
@@ -93,6 +92,13 @@ export const Sandpack: SandpackInternal = (props) => {
     height: props.options?.editorHeight, // use the original editor height
   };
 
+  const RightColumn: React.FC<{ children: React.ReactNode }> = ({ children }) =>
```

Comment:
Hummm... I'm afraid that this will remount everything and its children on every render. 

What about conditionally passing the props to the `RightColumn`? Something like this:

```jsx
const rightColumnProps = useMemo(() => hasRightColumn ? { style } : {}, [hasRightColumn])

<RightColumn {...rightColumnProps}>
```

Reply:
Wow yep, I wasn't thinking, haha. Updated 👍